test/models/merchant_test.rb

```diff
@@ -1,7 +1,27 @@
 require "test_helper"
 
 describe Merchant do
-  # it "does a thing" do
-  #   value(1+1).must_equal 2
-  # end
+  
+  describe "Member Since Method" do
+    # it "can calculate how long a merchant has been a member" do
```

Comment:
Test is commented out.  Was working fine until order was added over the weekend.  I'm going to work on this after the merge.